ALU_wrapper.py

```diff
@@ -0,0 +1,128 @@
+#!/apps/bio/software/anaconda2/envs/hcp/bin/python3
+
+# Please note, this is an early version. All the verbose-stuff will be reduced at later stages.
+# TODO: functions for copy finished data to web folder
+# TODO: Information to slims?
+# TODO: What to do with original folders?
+
+import os
+import glob
+import smtplib
+import argparse
+import subprocess
+from email.message import EmailMessage
+
+mail_to = ""  # Change this, and uncomment argument before EOF
+pythonpath = "/apps/bio/software/anaconda2/envs/hcp/bin/python3"  # since server (as for now) has python2 installed
```

Comment:
I again think this could be controlled by having a conda env

Reply:
This should be removed.. I missed that, but this is not used in this version of the script :|